build/write-image.sh

```diff
@@ -72,13 +74,13 @@ sudo cp -R frontend/dist/ui /tmp/eos-mnt/var/www/html/main
 sudo mkdir -p /tmp/eos-mnt/root/.ssh
 
 # Custom MOTD
-sudo sed -i 's/ENABLED=1/ENABLED=0/g' /tmp/eos-mnt/etc/default/motd-news
-sudo rm /tmp/eos-mnt/etc/update-motd.d/50-landscape-sysinfo
-sudo cp ./build/00-embassy /tmp/eos-mnt/etc/update-motd.d
-sudo chmod -x /tmp/eos-mnt/etc/update-motd.d/*
-sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/00-embassy
-sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/90-updates-available
-sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/95-hwe-eol
+#sudo sed -i 's/ENABLED=1/ENABLED=0/g' /tmp/eos-mnt/etc/default/motd-news
+#sudo rm /tmp/eos-mnt/etc/update-motd.d/50-landscape-sysinfo
+#sudo cp ./build/00-embassy /tmp/eos-mnt/etc/update-motd.d
+#sudo chmod -x /tmp/eos-mnt/etc/update-motd.d/*
+#sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/00-embassy
+#sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/90-updates-available
+#sudo chmod +x /tmp/eos-mnt/etc/update-motd.d/95-hwe-eol
```

Comment:
IDK what the raspbian equivalent of this is,  so I've disabled it for now

@k0gen

Reply:
Done #1205